sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares.hpp

```diff
@@ -1,12 +1,22 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
 
+/**
+ * @brief Includes all public headers from Azure Storage Files Shares SDK library.
+ *
+ */
+
 #pragma once
 
+#include "azure/storage/files/shares/protocol/share_rest_client.hpp"
 #include "azure/storage/files/shares/dll_import_export.hpp"
 #include "azure/storage/files/shares/share_client.hpp"
+#include "azure/storage/files/shares/share_constants.hpp"
 #include "azure/storage/files/shares/share_directory_client.hpp"
+#include "azure/storage/files/shares/share_file_attributes.hpp"
 #include "azure/storage/files/shares/share_file_client.hpp"
 #include "azure/storage/files/shares/share_lease_client.hpp"
+#include "azure/storage/files/shares/share_options.hpp"
+#include "azure/storage/files/shares/share_responses.hpp"
 #include "azure/storage/files/shares/share_sas_builder.hpp"
 #include "azure/storage/files/shares/share_service_client.hpp"
```

Comment:
newly added headers shouldn't be exposed to customers directly.

Reply:
Thank you for the review! I'll undo changes to storage's include-all header included files. I'll leave the `ctx` rename.